lib/components/Dropdown.jsx

```diff
@@ -0,0 +1,80 @@
+/*
```

Comment:
Is it possible to use the Dropdown from shared instead?

Reply:
I was unable to get it to work because the Dropdown from shared behaves like an [uncontrolled component](https://reactjs.org/docs/uncontrolled-components.html). But here I want to change the elements and the selected item from the outside, which is harder with an uncontrolled component.